addon/components/paper-dialog-inner.js

```diff
@@ -4,10 +4,10 @@ import PaperDialog from './paper-dialog';
 
 const { Component, computed } = Ember;
 
-export default Component.extend(Translate3dMixin, {
+export default Component.extend(/*Translate3dMixin, TODO */ {
   tagName: 'md-dialog',
-  classNames: ['md-default-theme'],
-  classNameBindings: ['contentOverflow:md-content-overflow', 'fullscreen:md-dialog-fullscreen'],
+  classNames: ['md-default-theme'     , 'md-transition-in'     /* TODO */ ],
+  classNameBindings: ['contentOverflow:md-content-overflow', 'dialogComponent.fullscreen:md-dialog-fullscreen'],
```

Comment:
Can't we get rid of `dialogComponent` properties by invoking `paper-dialog-inner` as
```hbs
{{#paper-dialog-inner fullscreen=fullscreen clickOutsideToClose=clickOutsideToClose ...
```
?

Since `paper-dialog-inner` is a private component, we can use it like we want to. No need for a `nearestOfType` dependency, probably.